src/appengine/handlers/upload_testcase.py

```diff
@@ -221,10 +233,14 @@ def get(self):
 
     result, params = get_result(self)
     self.render(
-        'upload.html', {
+        'upload.html',
+        {
             'fieldValues': {
+                'blackboxFuzzers':
+                    filter_blackbox_fuzzers(allowed_fuzzers),
                 'jobs':
                     allowed_jobs,
+                # TODO(unassigned): Add honggfuzz.
```

Comment:
lets just add it, it is 2 line change here and another 2 line one in appengine/private/components/upload-testcase/upload-form.html

Reply:
Done!